src/module.ts

```diff
@@ -1,11 +1,61 @@
-import { AppPlugin } from '@grafana/data';
+import { AppPlugin, PluginMeta } from '@grafana/data';
 import { GlobalSettings } from './types';
 import { App } from 'components/App';
 import { ConfigPage } from 'page/ConfigPage';
+import { findSMDataSources } from 'utils';
+import { getBackendSrv } from '@grafana/runtime';
+import { updateSMDatasource } from 'initialization-utils';
+import { getDashboardsNeedingUpdate, importAllDashboards } from 'dashboards/loader';
 
 export const plugin = new AppPlugin<GlobalSettings>().setRootPage(App).addConfigPage({
   title: 'Config',
   icon: 'cog',
   body: ConfigPage,
   id: 'config',
 });
+
+const getPluginSettings = async () =>
```

Comment:
We have to fetch this from the backend because `plugin.meta` is undefined at this point in the execution